core/src/it/scala/org/ensime/core/RefactoringHandlerSpec.scala

```diff
@@ -7,21 +7,28 @@ import java.nio.charset.Charset
 import java.text.SimpleDateFormat
 import java.util.Date
 
+import scala.concurrent.ExecutionContext.Implicits.global
+
 import org.ensime.api._
 import org.ensime.fixture._
 import org.ensime.util.EnsimeSpec
 import org.scalatest.Assertions
-import scala.concurrent.ExecutionContext.Implicits.global
 
 class RefactoringHandlerSpec extends EnsimeSpec
     with IsolatedAnalyzerFixture
     with RichPresentationCompilerTestUtils
     with RefactoringHandlerTestUtils {
 
   val encoding = "UTF-16"
-  def original = EnsimeConfigFixture.EmptyTestProject.copy(
-    compilerArgs = List("-encoding", encoding)
-  )
+  def original = {
+    val empty = EnsimeConfigFixture.EmptyTestProject
+    val origScalacOptions = empty.projects.head.scalacOptions
+    empty.copy(
+      projects = List(empty.projects.head.copy(
+        scalacOptions = origScalacOptions.patch(origScalacOptions.indexOf("-encoding"), List("-encoding", encoding), 2)
```

Comment:
this is not what you mean to do

```
List("foo", "bar", "baz", "gaz").patch(-1, List("FOO", "BAR"), 2)
```

you're erasing the first two elements if there is no match for `-encoding`.

Just add to the end of the original list, no need to be fancy.